### PR TITLE
aver-hid: Add support for AVer VB342 Pro

### PR DESCRIPTION
**Pull request by luswdev**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [x] Feature
- [ ] Documentation

aver-hid: Add support for AVer VB342 Pro

### CODE REVIEW

File: plugins/aver-hid/fu-aver-hid-device.c [hunk 1]
```diff
@@ -80,6 +81,18 @@ fu_aver_hid_device_poll(FuDevice *device, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_aver_hid_device_check_dual_isp(FuAverHidDevice *device)
```
Comment (hughsie) [inline plugins/aver-hid/fu-aver-hid-device.c:85]: I think a much better way to do this is to use:

    # AVer VB342Pro
    [USB\VID_2574&PID_0AB1]
    Plugin = aver_hid
    Flags = dual-isp

and then use `fu_device_register_private_flag` in the plugin, and then check something like `fu_device_has_private_flag(device, FU_AVER_HID_FLAG_DUAL_ISP)` to check that flag.

File: plugins/aver-hid/fu-aver-hid-device.c [hunk 2]
```diff
@@ -160,9 +173,15 @@ fu_aver_hid_device_isp_file_dnload(FuAverHidDevice *self,
 			return FALSE;
 
 		/* copy in payload */
-		fu_struct_aver_hid_req_isp_file_dnload_set_custom_isp_cmd(
-		    req,
-		    FU_AVER_HID_CUSTOM_ISP_CMD_FILE_DNLOAD);
+		if (fu_aver_hid_device_check_dual_isp(self) == TRUE) {
```
[the remark this answers is not in the document]
Comment (luswdev) [reply plugins/aver-hid/fu-aver-hid-device.c]: I've change to `fu_device_has_private_flag`

**Comment by hughsie** [comment]
@luswdev I see:

    - hook id: clang-format
    - exit code: 1

Can you try running `find . -name '*.c'| xargs clang-format -i` when in the `plugins/aver-hid` directory and then committing the result please. Then LGTM.

**Comment by luswdev** [comment]
Recommited _plugins/aver-hid/fu-aver-hid-device.c_ with clang format.

**Review by hughsie — APPROVED** [review]

**Pull request merged**